.azure-pipelines/client.yml

```diff
@@ -39,7 +39,18 @@ jobs:
           version: 2.1.503
 
       - task: DotNetCoreCLI@2
-        displayName: 'Build & Test'
+        displayName: 'Build & Test (no live tests)'
```

Comment:
Looks like this was only added for the Windows leg. We will need to do the same in the Linux one

Reply:
Indeed -- I didn't page down in the file 🤦‍♂️